pandas/core/categorical.py

```diff
@@ -1862,8 +1990,11 @@ def convert_categorical(x):
     if get_dtype_kinds(to_concat) - set(['object', 'category']):
         # convert to object type and perform a regular concat
         from pandas.core.common import _concat_compat
-        return _concat_compat([np.array(x, copy=False, dtype=object)
-                               for x in to_concat], axis=0)
```

Comment:
[docformatter](https://pypi.python.org/pypi/docformatter) is useful for fixing strings and docstrings

That said, I'm not sure what a correct pep8 thing is for these long ValueError / warns (I'm confident the rest can be done with yapf knobs). Perhaps we could increase the line width :suspect: jk